cachix-api/src/Cachix/Types/BinaryCache.hs

```diff
@@ -11,6 +11,10 @@ data BinaryCache = BinaryCache
     isPublic :: Bool,
     publicSigningKeys :: [Text],
     githubUsername :: Text,
-    permission :: Permission
+    permission :: Permission,
+    compression :: CompressionMode
```

Comment:
I'm wondering whether we should provide some sort of backward compatibility for this, i.e. maybe this value is stored in some sort of DB, and old values may failure to decode

Reply:
Domen told me, that he will take this part.